CONTRIBUTING.md

```diff
@@ -1,38 +1,7 @@
-<!-- TOC depthFrom:1 depthTo:6 withLinks:1 updateOnSave:1 orderedList:0 -->
+# Contributor Guidelines
+If you plan to make your first contribution, please start with the [**Contributor Guide**](https://github.com/openintegrationhub/BusinessCommittee/tree/master/Contributing/Guide) for general information about making contributions to the Open Integration Hub project.
 
-- [General Workgroup Information](#general-workgroup-information)
-	- [Roles](#roles)
-		- [Manager](#manager)
-		- [Committers](#committers)
-		- [Contributors](#contributors)
-- [Decisions](#decisions)
-- [Naming Conventions](#naming-conventions)
-
-<!-- /TOC -->
-
-# General Workgroup Information
-- Each workgroup has at least one **status call** every two weeks
-- **Every Committer** must attend the status call
-
-## Roles
-### Manager
-- Each workgroup has one manager - this manager is elected every 6 month by all committers within the workgroup (re-election possible)
-- The manager moderates discussions, documents decisions and coordinates activities, content and appointments
-- Only the manager is allowed to merge content into the master branch
-- Is part of one of the comittees
-
-### Committers
-- Decisions within a workgroup are made by the committers on democratic basis (unanimously)
-- Committers can choose which contributors shall become committers (Committees have to agree on choice)
-- Need to actively contribute to the workgroup
-
-### Contributors
-- Contributors contribute to the projects but do not have a vote
-- Contributors can apply to get a vote and become a Committer
-
-# Decisions
-Decisions within a workgroup are made by all committers (unanimously).
-Íf it is not possible to decide on a certrainissue/topic it must be escalated to the superordinated committee.
+The following represent additional rules specifically for contributions in this workgroup and repository.
```

Comment:
I can't see any following information.

Reply:
The naming convention below. Its the only specific information not mentionend in the overall guide. Other repros like openintegration hub have more, but here its just one.